tests/test_data/core/core__count_patient.txt

```diff
@@ -1,12 +1,12 @@
-(17, None, 'male', 'white', 'not hispanic or latino')
```

Comment:
this is removing the patient age field (which looks like it didn't work well with our synthetic data anyway). This is now only calculated in encounters relative to the age at the time of visit.